### DISCUSSION TITLE
Issues signing in

### DISCUSSION THREAD

**Discussion by Nicola17** [esa/SpOC2#10] [General]
I've registered to the optimise portal, and I've received a validation email on March 20. The validation email did not work, and when I try to ask for a new password, the site does not send anything to my email address.

Any chance that I can somehow recover my account without the need to create a new one?

**Comment by EmBlazquez** [comment]
Hi @Nicola17 ,

Mmh this is weird your appears as verified in our system. I tried resetting the validation, can you please give it another try? If it doesn't work I will delete your account and you will need to create another one ... sorry for that.

**Reply by Nicola17** [reply]
Unfortunately I cannot login and when I request to reset the pwd the email never arrives. If you can cancel the account that would be great!

**Reply by EmBlazquez** [reply]
@Nicola17 account canceled!

**Comment by Nicola17** [comment]
Now it works!

On Tue, Apr 11, 2023 at 6:46 PM Emmanuel Blazquez ***@***.***>
wrote:

> @Nicola17 <https://github.com/Nicola17> account canceled!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/esa/SpOC2/discussions/10#discussioncomment-5583286>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABFQ75O3THJKDABLCCLDDHDXAWDGZANCNFSM6AAAAAAWZLYOLA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 


Nicola Pezzotti
